runs/2018/mipyt-zima/info.yml

```diff
@@ -0,0 +1,57 @@
+title: MI-PYT (Pokročilý Python)
+subtitle: ČVUT - zimní semestr 2017/2018
+time: 9:15 a 11:45
+place: Učebna T9:303
+description: Poznej zajímavé knihovny Pythonu
+long_description: |
+    Cílem předmětu je naučit se různé pokročilé techniky a postupy programování v jazyce Python.
+    Předmět nepřímo navazuje na Programování v Pythonu (BI-PYT).
+    Předmět je zaměřen prakticky a má pouze cvičení, vše je prezentováno na příkladech.
+    Hodnocení je založeno na práci na cvičeních a semestrální práci.
+    Výuka předmětu probíhá pod vedením pracovníků z firmy Red Hat.
+vars:
+  coach-present: true
+  mi-pyt: true
+derives: mi-pyt
+plan:
+- base: requests-click
+  date: 2018-10-03
+
+- base: flask
+  date: 2018-10-10
+
+- base: pandas
+  date: 2018-10-17
+
+- base: distribution
+  date: 2018-10-24
+
+- base: numpy
+  date: 2018-10-31
+
+- base: testing
+  date: 2018-11-07
+
+- base: cython
+  date: 2018-11-14
+
+- base: docs
+  date: 2018-11-21
+
+- base: qt
+  date: 2018-11-28
+
+- base: async
+  date: 2018-12-05
+
+- base: magic
+  date: 2018-12-12
+
+- base: upy
+  date: 2018-12-19
```

Comment:
@encukou Would you be able to come? It's Pyvo date. We _need you_ (and your hardware) for MicroPython and we can possibly shuffle around async/magic/upy.

Reply:
Yes.
30 hardwares, right?

Reply:
24 working ones would do. 30 if we include some extras for sure.